.rvmrc

```diff
@@ -4,7 +4,7 @@
 # development environment upon cd'ing into the directory
 
 # First we specify our desired <ruby>[@<gemset>], the @gemset name is optional.
-environment_id="ruby-1.9.2-p290"
+environment_id="ruby-2.0.0-p451"
```

Comment:
If you're going to mess with this, just rm it and put in a .ruby-version (which is the common standard)